src/sage/groups/matrix_gps/finitely_generated_gap.py

```diff
@@ -120,7 +120,7 @@ def as_permutation_group(self, algorithm=None, seed=None):
             21499084800
             sage: P = G.as_permutation_group()
             sage: Psmaller = G.as_permutation_group(algorithm="smaller", seed=6)
-            sage: P == Psmaller
+            sage: P == Psmaller # skip_conda Fails for some reason when executed under conda
```

Comment:
This is fixed in https://github.com/sagemath/sage/pull/36792 with a plausible explanation.